src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -150,6 +151,51 @@ async function getAdditionalFacilityInfo(futureAppointments, useV2 = false) {
   return facilityData;
 }
 
+/**
+ * Wrapper function to retrieve facility information from the appointment
+ * record when using the v2 api or make api call to retrieve facility
+ * information.
+ *
+ * @param {*} appointments
+ * @param {*} dispatch
+ * @param {*} featureFacilitiesServiceV2
+ */
+async function getAdditionalFacilityInfoV2(
+  appointments,
+  dispatch,
+  featureFacilitiesServiceV2,
+) {
+  let facilityData;
+  if (featureFacilitiesServiceV2) {
+    // Facility information included with v2 appointment api call.
+    const facilities = appointments
+      .map(
+        appt =>
+          appt.vaos.apiData.location
+            ? { ...appt.vaos.apiData.location.attributes }
```

Comment:
I think you could return `appt.vaos.apiData.location.attributes` directly, rather than do the `{... }` part. That's creating a shallow copy of the attributes, which I don't think is necessary, since we don't modify them in the transformer.